udata/tests/api/test_user_api.py

```diff
@@ -161,15 +161,16 @@ def test_suggest_users_api_by_id(self):
         with self.autoindex():
             users = UserFactory.create_batch(4)
 
-        user = users[0]
+        first_user = users[0]
         response = self.get(url_for('api.suggest_users'),
-                            qs={'q': str(user.id), 'size': '5'})
+                            qs={'q': str(first_user.id), 'size': '5'})
         self.assert200(response)
 
-        self.assertGreaterEqual(len(response.json), 1)
-
-        suggestion = response.json[0]
-        self.assertEqual(suggestion['id'], str(user.id))
+        # The batch factory generates ids that might be too close
+        # which then are found with the fuzzy search.
+        suggested_ids = [u['id'] for u in response.json]
+        self.assertGreaterEqual(len(suggested_ids), 1)
+        self.assertIn(str(first_user.id), suggested_ids)
```

Comment:
You choose to keep the fuzzy search, OK 👍
(We will have to this carefully on completer to ensure fuzzy does not create too much noise)